### ISSUE TITLE
Gilded Netherite & Enderite Armor support

### ISSUE THREAD

**Issue by nco2k** [A5b84/armor-chroma-fabric#21]
would be cool if this mod supported [Gilded Armor](https://modrinth.com/mod/gildedarmor) & [Enderite Mod](https://modrinth.com/mod/enderite-mod).

**Comment by Progical** [comment]
I too would like to see expanded mod compatibility, which is why I spent all day figuring out how to do it myself.
![Screenshot 2023-11-03 225216](https://github.com/A5b84/armor-chroma-fabric/assets/32730946/4125c040-028c-4b51-8683-785fdec7348f)
So far the list includes:
- Additional Additions (Updated/ RP Support)
- Dragon Loot (RP Support)
- Tech Reborn (Updated/ RP Support)
- Enderite (New)
- Gilded Armor (New)
- Galosphere (New)
- Full Turtle Armor (New)
I also wanted to add mod support for the included optional resource pack, which is what I did for the mods I decided to touch. This mod is one I think would benefit greatly from a large mod compatibility (which I'd like to contribute to). I'm not quite sure how to use GitHub, but perhaps I can get these and more into the main branch? Just gotta figure out more mods people would like to see supported.

**Comment by nco2k** [comment]
> This mod is one I think would benefit greatly from a large mod compatibility

agreed. this is a must-have mod imho and would greatly benefit from supporting more mods. it should also support armor from the BetterNether and BetterEnd mods, because they are very popular.

**Comment by A5b84** [comment]
Hi

As discussed somewhere around [here](https://github.com/A5b84/armor-chroma-fabric/issues/25#issuecomment-2471664219), if you publish your icons as a resource pack I'd be happy to to add a link to it in the mod's description

And for future reference, I've added some documentation [here](https://github.com/A5b84/armor-chroma-fabric/blob/main/docs/resource-packs.md)
